output-guidelines/micropublications.md

```diff
@@ -1,7 +1,70 @@
+
+### Title
+A single line which provides the gist of the research conducted
+  
+### Type of research
+Each micropublication should provide evidence to support **one** specific:
+  - claim,
+  - observation, or
+  - method
+
+Please choose one of the above 3 options.
+
+### A summary for the general public
+All teams are expected to submit a short summary of their research that can be easily grasped by a non-scientific audience. This summary will be shared on our website and social media to make your research more accessible and to help your research have a larger *impact*. You are welcome to submit a text summary if you'd like but we are not providing any prescriptions for what this summary should look like as we'd like the scholars to be able to use any creative means (such as videos, comics, poems, songs) to communicate their research findings. 
+
+- Format : please avoid using proprietary data formats for your creative submissions. 
+- For inspiration, checkout some of the following resources:
+  - [Cartoonists tackling climate change through webcomics](https://www.dw.com/en/cartoonists-tackling-climate-change-through-webcomics/a-63499432)
+  - [Rajan lab comics](https://www.rajanlab.com/comics)
+  - [Dance your PhD](https://www.youtube.com/results?search_query=dance+your+phd)
+
+### Author Contributions
+We request all the teams to track author contributions using the [CRediT taxonomy](https://credit.niso.org). Click on each of the 14 contributor roles to see what they constitute. We discourage giving credits to a scholar (including any former TAs) or mentor if they haven't contributed to the project at all. 
+- Format: .csv, .ods, or .tsv
+- Templates: We are providing 2 templates to track author contributions throughout the program. **Each team is required to submit author contributions in one of these 2 formats.** Download one of the following templates for your team:
+  - [CISP_contributors_table_template](https://docs.google.com/spreadsheets/d/140bNO83-GkOP1SYXWLpZmY27a5wxtj97gOR-5CaMXDQ/edit?usp=sharing): This template was adopted from [tenzing.club](https://contributorshipcollaboration.github.io/projects/tenzingclub). Feel free to leave blanks for Roles that aren't relevant for your team. Please use the Notes column to let us know of any specifics that aren't captured in the 14 categories (such a equal contribution, or additional roles other than the 14 that are a part of CRediT).
+  - [CISP_contributors_table_template_granular](https://docs.google.com/spreadsheets/d/1fw34ZjKaZQ9iv6ld6YT1kZP2uyg0CbmJIZdM88tb07E/edit?usp=sharing): This is a replica of the previous template with more granularity added for each role. There are 2 types of contributions to choose from:
+    - Lead: If a scholar has made major contributions in a role, choose this option.
+    - Support: If a scholar has primarily provided support and feedback or helped with brainstorming but hasn't made major contributions to the role, choose this option.
+   
```

Comment:
```suggestion
Feel free to leave blanks for Roles that aren't relevant for your team. Please use the Notes column to let us know of any specifics that aren't captured in the 14 categories, such as additional roles or other degrees of contribution.
```

Reply:
Added this here and made a small change as I think it could be confusing to mention "equal" contributions by name if we are using "lead" to indicate major (even if equal) contributions in the more granular template despite the CRediT taxonomy having three degrees of granularity: lead, equal, and support. To be clear, I am definitely in favour of using only two degrees since the terms "lead" and "equal" can be ambiguous too. See the [fair criticism](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0244611) from the tenzing.club authors for example.

>  It seems likely that if the “equal” degree is used, it must be used for multiple co-authors as it may not make sense when applied to just one. This is not currently addressed, however, by the CRediT documentation, nor are other possible constraints such as whether “equal” can be used as an intermediate indicator in cases where there are already authors with the “lead” and “supporting” labels.